ambry-rest/src/main/java/com.github.ambry.rest/NettySslFactory.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.rest;
```

Comment:
should this also be in commons ? (I know that would introduce a dependence on netty in commons)

Reply:
I am okay with this. I suppose netty libraries can be useful in other places in the future. We could also keep it in rest for now and move it if it seems like it could be useful on the server side too?